src/main/java/com/flywithingryd/IngrydAirways/model/Itinerary.java

```diff
@@ -0,0 +1,19 @@
+package com.flywithingryd.IngrydAirways.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity(name = "itinerary")
+public class Itinerary {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    private Passenger passenger;
+    private Reservation reservation;
```

Comment:
This would be a list of reservation as an itinerary can have many reservations within